src/tasks/entities.cr

```diff
@@ -191,17 +191,18 @@ module PlaceOS::Tasks::Entities
       new_driver
     end
 
-    zone = upsert_document(Model::Zone.all) do
-      zone_name = "TestZone-#{version}"
-      new_zone = Model::Zone.new(name: zone_name)
-      new_zone
+    # Create default entities for each element of the hierarchy
+    zones = {"building", "org", "level", "area"}.map do |tag|
+      upsert_document(Model::Zone.with_tag(tag)) do
+        Model::Zone.new.tap do |zone|
```

Comment:
Should these be setting up the `parent` hierarchy too?

Reply:
Purpose of this is for configuring `source`, parent association isn’t strictly required for the development environment. These entities are ignored when the env is `production`